website/pages/docs/agent/template/index.mdx

```diff
@@ -20,8 +20,7 @@ The `template` stanza configures the templating engine in the Vault agent for re
 secrets to files using Consul Template markup language. Multiple `template` stanzas
 can be defined to render multiple files.
 
-When the agent is started with templating enabled, it will attempt to acquire a
-Vault token using the configured Method. On failure, it will back off for a short
+When the agent is started with templating enabled, it will attempt to acquire a Vault token using the configured Method. On failure, it will back off for a short
```

Comment:
Is this change intentional?

Reply:
Nope. I've reverted it in my local branch and as soon as I've added some more tests I'll push the new branch up and update this PR